lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -98,6 +98,7 @@ TF_DEFINE_PUBLIC_TOKENS(MayaUsdProxyShapeBaseTokens,
 MayaUsdProxyShapeBase::ClosestPointDelegate
 MayaUsdProxyShapeBase::_sharedClosestPointDelegate = nullptr;
 
+const auto kanonymousLayerName{"anonymousLayer1"};
```

Comment:
createInMemory takes a std::string, so instead of auto why not make this a std::string? 

According to our coding standards constants "should be named with a leading “k” followed by UpperCamelCase". So this should be "kA..."

Reply:
auto already deduces the type to std::string.

Reply:
@seando-adsk please see 5dd82214fad78999da9c022c81cb6d2e588356d3